web/themes/custom/novel/templates/fields/field--node--field-ticket-categories--event.html.twig

```diff
@@ -0,0 +1,8 @@
+{% if items|length == 1 %}
+  {# Events with a single ticket category should only show the price. #}
+  <span>{{ items.0.content['#paragraph'].field_ticket_category_price|view }}</span>
```

Comment:
i think you can just do

```suggestion
  <span>{{ entity_type. field_ticket_category_price.0 }}</span>
```

Reply:
or, alternatively: (This is just mock code, it probably doesnt work, but i hope you get the idea)

```
{% set view_mode = (items|length == 1) ? 'default' : 'single' %}


  {% for item in items %}
    {{ item.content|view(view_mode) }}
  {% endfor %}

Reply:
The first approach does not work. It will just render the first ticket category with both name and price.

I get the idea of your suggestion about introducing a new `single` view mode but it gets quite specialized. It will also force us to consider this view mode for other paragraphs where I do not think it will be relevant.